apis/infix-en_GB/atrium-api-infix-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/MapAssertionsSpec.kt

```diff
@@ -0,0 +1,240 @@
+import ch.tutteli.atrium.specs.*
+
+class MapExpectionsSpec : ch.tutteli.atrium.specs.integration.MapAssertionsSpec(
+    fun2(Companion::contains),
+    fun2(Companion::contains).name to Companion::containsNullable,
+    "${fun2(Companion::contains).name} ${KeyValue::class.simpleName}" to Companion::containsKeyWithValueAssertions,
+    "${fun2(Companion::contains).name} ${KeyValue::class.simpleName}" to Companion::containsKeyWithNullableValueAssertions,
+    fun1(Companion::containsKey),
+    fun1(Companion::containsNullableKey),
+    fun1(Companion::containsNotKey),
+    fun1(Companion::containsNotNullableKey),
+    /* string toBe, notToBe to avoid ambiguity error */
+    "toBe ${Empty::class.simpleName}" to Companion::isEmpty,
+    "notToBe ${Empty::class.simpleName}" to Companion::isNotEmpty
+) {
+    companion object {
+        private fun contains(plant: Expect<Map<out String, Int>>, pair: Pair<String, Int>, otherPairs: Array<out Pair<String, Int>>): Expect<Map<out String, Int>> {
+            return if (otherPairs.isEmpty()) {
+                plant contains (pair.first to pair.second)
+            } else {
+                plant contains Pairs(pair, *otherPairs)
+            }
+        }
+
+        private fun containsNullable(plant: Expect<Map<out String?, Int?>>, pair: Pair<String?, Int?>, otherPairs: Array<out Pair<String?, Int?>>): Expect<Map<out String?, Int?>> {
+            return if (otherPairs.isEmpty()) {
+                plant contains (pair.first to pair.second)
+            } else {
+                plant contains Pairs(pair, *otherPairs)
+            }
+        }
+
+        private fun containsKeyWithValueAssertions(plant: Expect<Map<out String, Int>>, keyValue: Pair<String, Expect<Int>.() -> Unit>, otherKeyValues: Array<out Pair<String, Expect<Int>.() -> Unit>>) : Expect<Map<out String, Int>> {
+            return if (otherKeyValues.isEmpty()) {
+                plant contains KeyValue(keyValue.first, keyValue.second)
+            } else {
+                mapArguments(keyValue, otherKeyValues).to { KeyValue(it.first, it.second) }.let { (first, others) ->
+                    plant contains All(first, *others)
+                }
+            }
+        }
+
+        private fun containsKeyWithNullableValueAssertions(plant: Expect<Map<out String?, Int?>>, keyValue: Pair<String?, (Expect<Int>.() -> Unit)?>, otherKeyValues: Array<out Pair<String?, (Expect<Int>.() -> Unit)?>>): Expect<Map<out String?, Int?>> {
+            return if (otherKeyValues.isEmpty()) {
+                plant contains KeyValue(keyValue.first, keyValue.second)
+            } else {
+                mapArguments(keyValue, otherKeyValues).to { KeyValue(it.first, it.second) }.let { (first, others) ->
+                    plant contains All(first, *others)
+                }
+            }
+        }
+
+        private fun containsKey(plant: Expect<Map<out String, *>>, key: String)
+            = plant containsKey key
```

Comment:
could you please format the file with Intellij CTRL+ALT+L (the api-cc-infix uses an old formatting scheme which is no longer supported by intellij). Thanks